### PR TITLE
Fix #4155: Left panel filter does not reset after

### LINKED ISSUE #4155: Left panel filter does not reset after navigating to different entity tab
**Affected module**
UI

**Describe the bug**
The left panel filter does not reset after navigating to a different entity tab

**To Reproduce**
- Navigate to Tables on explore page
- Click on any filter from the left panel
- Navigate to Dashboards
- A message is displayed "No matching data assets found"
- Navigate back to the Tables page
- Filter is not cleared before navigating to a different tab

https://user-images.githubusercontent.com/67048554/163466507-8786826a-2109-4957-a547-dc9be3634048.mp4

**Expected behavior**
Filter should be cleared before navigating to different tab

**Version:**
 - OpenMetadata version: 0.9.0

**Additional context**
NA

### PR DESCRIPTION
**Pull request by darth-coder00**
Closes #4155 
### Describe your changes :

I worked on fixing Explore page filter persistence and reset

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
<p align="center">

https://user-images.githubusercontent.com/86726556/166266115-677ffe53-51d9-4c7a-bb17-35ae7db2fec2.mov

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui @harshach

### CODE REVIEW

**Review by vivekratnavel — APPROVED** [review]

**Comment by vivekratnavel** [comment]
@kshinde2512 The following tests are failing:
``` 
Results:
2022-05-02T16:42:30.1106836Z [INFO] 
2022-05-02T16:42:30.1107104Z [ERROR] Errors: 
2022-05-02T16:42:30.1117474Z [ERROR]   Explore.checkDashboardCount:158 » IndexOutOfBounds Index 1 out of bounds for l...
2022-05-02T16:42:30.1118398Z [ERROR]   Explore.checkPipelineCount:185 » IndexOutOfBounds Index 1 out of bounds for le...
2022-05-02T16:42:30.1119139Z [ERROR]   Explore.checkTableCount:104 » IndexOutOfBounds Index 2 out of bounds for lengt...
2022-05-02T16:42:30.1119910Z [ERROR]   Explore.checkTopicCount:131 » IndexOutOfBounds Index 1 out of bounds for lengt...
```

File: openmetadata-ui/src/main/resources/ui/src/pages/tour-page/TourPage.component.tsx [hunk 1]
```diff
@@ -70,6 +70,14 @@ const TourPage = () => {
     setExplorePageCounts(exploreCount);
   };
 
+  /**
+   *
+   * @returns void
+   */
+  const handleFilterChange = () => {
```
Comment (vivekratnavel) [inline openmetadata-ui/src/main/resources/ui/src/pages/tour-page/TourPage.component.tsx:77]: Why do we need this?

Comment (darth-coder00) [reply openmetadata-ui/src/main/resources/ui/src/pages/tour-page/TourPage.component.tsx:77]: Just an empty callback handler for explore component as it's not in use for tours page, but this prop is mandatory

**Comment by darth-coder00** [comment]
Failing Selenium tests are not related to the changes and are fixed in PRs
- #4576
- #4645

**Review by shahsank3t — APPROVED** [review]

**Pull request merged**